contracts/src/Teleporter/tests/RedeemRelayerRewardsTests.t.sol

```diff
@@ -84,6 +84,12 @@ contract RedeemRelayerRewardsTest is TeleporterMessengerTest {
                 (feeRewardInfo.relayerRewardAddress, feeRewardInfo.feeAmount)
             )
         );
+        vm.expectEmit(true, true, true, true, address(teleporterMessenger));
```

Comment:
do we need to check for all topics if the event is only indexing topic 1

Reply:
We don't necessarily _need_ to, but I think it is safest to always check for all topics to match by default. The only case where this isn't possible is if the topic value for a specific log emission is not known beforehand, but otherwise I would rather any changed behavior be more likely to be caught by the units tests.

Reply:
agreed, we should check all topics for the `expectEmit` in case of future changes.